.gitignore

```diff
@@ -147,8 +147,5 @@ export-env-vars-and-run-mhs-docker.sh
 # system
 .DS_Store
 
-# enviornment file
-integration-tests/integration_tests/integration-tests-env.yaml
```

Comment:
In previous PR you removed it from `# Local configuration` and added here. I wrote to leave it there in `# Local configuration`. Here I see you are rolling back half of your change

Reply:
ahh sorry, I didn't see it's there